.changes/add-proxy-for-nsis-download.md

```diff
@@ -0,0 +1,5 @@
+---
+"tauri-bundler": 'patch:fix'
+---
+
+Add proxy for NSIS when downloading resources.
```

Comment:
change tags are defined in `.changes/config.json` and so the correct tag here would be `bug`
```suggestion
"tauri-bundler": 'patch:bug'

Read `HTTP_PROXY` env var when downloading bundling resources on Windows.
```